src/main/scala/org/broadinstitute/dsde/firecloud/service/SubmissionService.scala

```diff
@@ -18,36 +18,43 @@ trait SubmissionService extends HttpService with PerRequestCreator with FireClou
         passthrough(submissionQueueStatusUrl, GET)
       }
     } ~
```

Comment:
Sorry that the diff below looks painfully difficult to parse. All I did was turning `"workspaces" / Segment / Segment / "submissions"` prefix into `"workspaces" / Segment / Segment` and moving `submissions` underneath along with the newly added `submissionsCount`.